src/i18n/fi/translation.json

```diff
@@ -140,6 +141,7 @@
       "title": "Ammatilliset erityisosaamiset"
     }
   },
+  "log-in-for-favorites": "Kirjaudu sisään Osaamispolkuun, jotta voit tallentaa mahdollisuuksia suosikkeihin ja päästä tarkastelemaan niitä myöhemmin.",
```

Comment:
Maybe this should be login-for-favorites? 
Wording could be more generic e,g, "Kirjaudu sisään Osaamispolkuun, jotta voit tallentaa suosikkeja ja päästä tarkastelemaan niitä myöhemmin."

Reply:
Changed. The wording was actually taken from Työmarkkinatori, but maybe more generic term works better here 🙂